backend/ops_api/ops/resources/portfolio_status.py

```diff
@@ -1,12 +1,28 @@
+from flask import Response, jsonify
+from flask_jwt_extended import jwt_required
 from models.base import BaseModel
+from models.portfolios import PortfolioStatus
 from ops_api.ops.base_views import BaseItemAPI, BaseListAPI
+from typing_extensions import override
 
 
 class PortfolioStatusItemAPI(BaseItemAPI):
     def __init__(self, model: BaseModel):
         super().__init__(model)
 
+    @jwt_required()
```

Comment:
You actually don't need this here, as it's inherited from the `BaseItemAPI`'s decorator.